ghcup-prereleases-0.0.7.yaml

```diff
@@ -636,6 +636,93 @@ ghcupDownloads:
+              dlUri: https://downloads.haskell.org/~ghc/9.6.0.20230210/ghc-9.6.0.20230210-i386-deb9-linux.tar.xz
+            unknown_versioning: *ghc-961alpha3-32-deb9
+          Linux_Mint:
+            unknown_versioning: *ghc-961alpha3-32-deb9
+          Linux_Ubuntu:
+            unknown_versioning: *ghc-961alpha3-32-deb9
+          Linux_UnknownLinux:
+            unknown_versioning: *ghc-961alpha3-32-deb9
+        A_64:
+          Darwin:
+            unknown_versioning:
+              dlHash: 25c32514e6aba55c510f01bb264d959a56c5fed114d7ef508de665c5eeaf47ed
+              dlSubdir: ghc-9.6.0.20230210-x86_64-apple-darwin
+              dlUri: https://downloads.haskell.org/~ghc/9.6.0.20230210/ghc-9.6.0.20230210-x86_64-apple-darwin.tar.xz
+          Linux_Alpine:
+            unknown_versioning:
+              dlHash: 5722aca0e4da55e1e05c20bfcd63d21c59d0800e1850e87b4b9ad0066a918279
+              dlSubdir: ghc-9.6.0.20230210-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.6.0.20230210/ghc-9.6.0.20230210-x86_64-alpine3_12-linux-static-int_native.tar.xz
+          Linux_CentOS:
+            ( >= 7 && < 8 ): &ghc-961alpha3-64-centos7
+              dlHash: 0f68876231e7bab3f261238533461aed0e231fa93b1483eef08a91dc510eddc1
+              dlSubdir: ghc-9.6.0.20230210-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.6.0.20230210/ghc-9.6.0.20230210-x86_64-centos7-linux.tar.xz
+            unknown_versioning: *ghc-961alpha3-64-centos7
+          Linux_Debian:
+            (>= 10 && < 11): &ghc-961alpha3-64-deb10
+              dlHash: fa0c2b8e31ea182f5111feeec3c3fcff2074857b5a068d8902519fdcc96b26e3
+              dlSubdir: ghc-9.6.0.20230210-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.6.0.20230210/ghc-9.6.0.20230210-x86_64-deb10-linux.tar.xz
+            < 10: &ghc-961alpha3-64-deb9
+              dlHash: 9a43cad78680d6a47e338ae5bc13deada164d7203996f7791e6957945c5aa5f3
+              dlSubdir: ghc-9.6.0.20230210-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.6.0.20230210/ghc-9.6.0.20230210-x86_64-deb9-linux.tar.xz
+            '>= 11': &ghc-961alpha3-64-deb11
+              dlHash: d552b709426dd8e1afa499789bd9ac775c1f062005656b4c41d28bb338d6692e
+              dlSubdir: ghc-9.6.0.20230210-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.6.0.20230210/ghc-9.6.0.20230210-x86_64-deb11-linux.tar.xz
+            unknown_versioning: *ghc-961alpha3-64-deb11
+          Linux_Fedora:
+            '>= 33': &ghc-961alpha3-64-fedora33
+              dlHash: ade217760de958e6ed8693b16245bd244a2ccf54c262b37222d784b839d8a699
+              dlSubdir: ghc-9.6.0.20230210-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.6.0.20230210/ghc-9.6.0.20230210-x86_64-fedora33-linux.tar.xz
+            unknown_versioning: *ghc-961alpha3-64-centos7
+          Linux_Mint:
```

Comment:
For both linux mint and ubuntu we want to use the respective bindists, see the previous mapping:

```yml
          Linux_Ubuntu:
            '< 19': &ghc-961alpha2-64-ubuntu18
              dlHash: 27e89a1210ac1ec434f947c635b1431cc19c8f15e0a1e0afa51d9356f87d3cb2
              dlSubdir: ghc-9.6.0.20230128-x86_64-unknown-linux
              dlUri: https://downloads.haskell.org/~ghc/9.6.0.20230128/ghc-9.6.0.20230128-x86_64-ubuntu18_04-linux.tar.xz
            unknown_versioning: &ghc-961alpha2-64-ubuntu20
              dlHash: 5115409cbec7c040345c7c4d335d48ee2789b05dc4669d95ccc609078f5d812a
              dlSubdir: ghc-9.6.0.20230128-x86_64-unknown-linux
              dlUri: https://downloads.haskell.org/~ghc/9.6.0.20230128/ghc-9.6.0.20230128-x86_64-ubuntu20_04-linux.tar.xz
          Linux_Mint:
            < 20: *ghc-961alpha2-64-ubuntu18
            '>= 20': *ghc-961alpha2-64-ubuntu20
```